go.mod

```diff
@@ -3,12 +3,12 @@ module github.com/schollz/hostyoself
 go 1.12
 
 require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/gorilla/websocket v1.4.0
 	github.com/h2non/filetype v1.0.8
-	github.com/jteeuwen/go-bindata v3.0.7+incompatible // indirect
```

Comment:
This was removed after running `go mod tidy`. 

I suspect this is because of how go modules work and the README's build instructions.

https://github.com/schollz/hostyoself/blob/04ad1997a612b36a11530f321060bd3e31b0b2c4/main.go#L3